README.md

```diff
@@ -11,6 +11,24 @@ Design document: https://docs.google.com/document/d/12gZow_JTJjRrmaD2mNTmYniLhyx
 [![GoDoc](https://godoc.org/github.com/celestiaorg/optimint?status.svg)](https://godoc.org/github.com/celestiaorg/optimint)
 [![Twitter Follow](https://img.shields.io/twitter/follow/CelestiaOrg?style=social)](https://twitter.com/CelestiaOrg)
 
+## Building From Source
+
+Requires Go version >= 1.17.
+
+To build:
+
+```sh
+git clone git@github.com:celestiaorg/optimint.git
```

Comment:
Using HTTPS URL here seems to be a bit more generic way (no need for GitHub SSH key management).

Reply:
Generally I am not sure how useful these instructions are. Optimint is made for go-devs who know how to clone and build code.

Reply:
Go doesn't have a single canonical way of building code, unlike Rust. Our repos in fact all use different variations of `make`, `make build`, `make install`, and `go build ./...`. I shouldn't have to dive into source code to figure out how to build.

Reply:
You'll have to look into the source code to make use of optimint anyways 🤷🏼 And optimint by itself is not useful without an ABCI app. I'll defer to Tomasz to decide though.

Reply:
IMHO `go build ./...` is the canonical way of building code.

I agree that this repo will be used only by devs, but even for external contributors, it might be helpful (you don't need to look for Makefile). Especially the testing part is important.